_studio/mfx_lib/plugin/include/mfx_vp9_dec_plugin.h

```diff
@@ -21,126 +21,23 @@
 #if !defined(__MFX_VP9_DEC_PLUGIN_INCLUDED__)
```

Comment:
Common practice to name such thing is __MFX_VP9_DEC_PLUGIN_H__, INCLUDED looks ugly

Reply:
Refactoring out of scope for this PR. Will consider alignment of include guards later.